.jenkins/Jenkinsfile

```diff
@@ -375,71 +377,89 @@ properties([buildDiscarder(logRotator(artifactDaysToKeepStr: '90',
+             "ACC1804 gcc Debug LVI" :                                  { ACCTest(AGENTS_LABELS["acc-ubuntu-18.04"], 'gcc', 'Debug', 'ControlFlow') },
+             "ACC1804 gcc Release LVI" :                                { ACCTest(AGENTS_LABELS["acc-ubuntu-18.04"], 'gcc', 'Release', 'ControlFlow') },
+             "ACC1804 Container RelWithDebInfo" :                       { ACCContainerTest(AGENTS_LABELS["acc-ubuntu-18.04"], '18.04') },
+             "ACC1804 Container RelWithDebInfo LVI" :                   { ACCContainerTest(AGENTS_LABELS["acc-ubuntu-18.04"], '18.04', 'ControlFlow') },
+             "ACC1804 Package RelWithDebInfo" :                         { ACCPackageTest(AGENTS_LABELS["acc-ubuntu-18.04"], '18.04') },
+             "ACC1804 Package RelWithDebInfo LVI" :                     { ACCPackageTest(AGENTS_LABELS["acc-ubuntu-18.04"], '18.04', 'ControlFlow') },
+             "ACC1804 GNU gcc SGX1FLC" :                                { ACCGNUTest() },
+             "AArch64 1604 GNU gcc Debug" :                             { AArch64GNUTest('16.04', 'Debug')},
+             "AArch64 1604 GNU gcc Release" :                           { AArch64GNUTest('16.04', 'Release')},
+             "AArch64 1804 GNU gcc Debug" :                             { AArch64GNUTest('18.04', 'Debug')},
+             "AArch64 1804 GNU gcc Release" :                           { AArch64GNUTest('18.04', 'Release')},
+             "Sim 1804 clang-7 SGX1 Debug" :                            { simulationTest('18.04', 'SGX1', 'Debug')},
+             "Sim 1804 clang-7 SGX1 Release" :                          { simulationTest('18.04', 'SGX1', 'Release')},
+             "Sim 1804 clang-7 SGX1-FLC Debug" :                        { simulationTest('18.04', 'SGX1FLC', 'Debug')},
+             "Sim 1804 clang-7 SGX1-FLC Release" :                      { simulationTest('18.04', 'SGX1FLC', 'Release')},
+             // "RHEL-8 simulation clang-8 SGX1 Release":             { RHEL8Test('clang', 'Release', ["OE_SIMULATION=1"]) }, // Enable when https://github.com/openenclave/openenclave/issues/2556 is fixed.
+             "RHEL-8 simulation clang-8 SGX1 Debug":                    { RHEL8Test('clang', 'Debug',   ["OE_SIMULATION=1"]) },
+             // "RHEL-8 simulation gcc-8 Release":                    { RHEL8Test('gcc',   'Release', ["OE_SIMULATION=1"]) }, // Enable when https://github.com/openenclave/openenclave/issues/2558 is fixed.
+             "RHEL-8 simulation gcc-8 SGX1 Debug":                      { RHEL8Test('gcc',   'Debug',   ["OE_SIMULATION=1"]) },
+             // "RHEL-8 ACC clang-8 Release" :                        { RHEL8Test('clang', 'Release') },                      // Enable when https://github.com/openenclave/openenclave/issues/2556 is fixed.
+             // "RHEL-8 ACC clang-8 Debug" :                          { RHEL8Test('clang', 'Debug') },                        // Enable when https://github.com/openenclave/openenclave/issues/2557 is fixed.
```

Comment:
The comment says to uncomment this line after #2557 is fixed, and that issue is marked as closed.